server/src/main/java/org/apache/druid/query/lookup/LookupIntrospectionResource.java

```diff
@@ -20,14 +20,17 @@
 package org.apache.druid.query.lookup;
 
 import com.google.inject.Inject;
+import com.sun.jersey.spi.container.ResourceFilters;
```

Comment:
This import is unused, and is probably related to the CI failure.

Reply:
This is actually being used for the ResourceFilters annotation below.